### PR TITLE
Updated version for new project idea

### PR DESCRIPTION
**Pull request by Gussamanel**
We have taken away godot completely and added what we discussed. @platisd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,16 @@
 
 
 # WHAT? 
-We are creating an application to drive the car maunally for seeking treasure(s). The car will be able to take multiple routes to find the treasure. The environment will consist of multiple objects/obstacles and one treasure. The user will be alerted when there is an object nearby and the car will be stopped if it gets too close to the object. The closer or further the car is from the treasure will be displayed on the app. The user will be able to view the car's speed and how much the car has travelled so far. The user will have a first person view (driver seat view and rear view). 
+We are creating an application to drive the car manually using a joystick. The car will be able to travel freely around the map. The user will be alerted when there is an object nearby through a pop up message on the app. If the car gets too close, the emergency break will be activated and the car will stop moving. The user will be able to view the car's speed and how much the car has travelled so far. The user will have a first person view (driver seat view and rear view).
```
Comment (platisd) [inline README.md:5]: You cannot have **two** cameras

**Review by jenniferhalgh — DISMISSED** [review]
Yes, this is what we discussed. Looks good!

**Review by platisd — CHANGES_REQUESTED** [review]
More feasible now, you can make things more advanced later if you want. A small remark.

File: README.md (hunk 1 above)
Comment (jenniferhalgh) [reply README.md:5]: Is it possible to switch views then? I.e not having two camera views at the same time but choosing which one should be activated and switch between them.

Comment (platisd) [reply README.md:5]: No

**Review by SadhanaAnandan — APPROVED** [review]
We have all the changes requested.

**Review by platisd — APPROVED** [review]

**Pull request merged**